src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <section class="block">
+      <div class="block__row-zero stars--0">
+        <div class="block__star"></div>
+        <div class="block__star"></div>
+        <div class="block__star"></div>
+        <div class="block__star"></div>
+        <div class="block__star"></div>
+      </div>
```

Comment:
```suggestion
      <div class="block__stars block__stars--0">
        <div class="block__star"></div>
        <div class="block__star"></div>
        <div class="block__star"></div>
        <div class="block__star"></div>
        <div class="block__star"></div>
      </div>
```
Тебе не нужен и модификатор block__row-zero, и stars--0. Оставь что-то одно и сделай по БЭМ. Модификатор пиши через "--", а не один дефис.